Sources/MockoloFramework/Utils/SwiftSyntaxExtensions.swift

```diff
@@ -0,0 +1,352 @@
+                    !name.hasPrefix(.underlyingVarPrefix) &&
+                    !name.hasSuffix(.closureVarSuffix) &&
+                    !name.hasSuffix(.callCountSuffix) &&
+                    !name.hasSuffix(.subjectSuffix) &&
+                    vtype != .unknownVal
+            }
+            
+            if let varmodel = Entity.model(name: name,
+                                           typeName: typeName,
+                                           acl: current?.acl,
+                                           isStatic: isStatic,
+                                           canBeInitParam: canBeInitParam,
+                                           offset: v.offset,
+                                           length: v.length,
+                                           modelDescription: node.description,
+                                           processed: current?.isProcessed ?? false) {
+                current?.members.append(varmodel)
+            }
+        }
+        return .visitChildren
+    }
+    
+    func visit(_ node: AssociatedtypeDeclSyntax) -> SyntaxVisitorContinueKind {
+        
+        var t = node.inheritanceClause?.inheritedTypeCollection.description ?? ""
+        t.append(node.genericWhereClause?.description ?? "")
+        
+        if let patmodel = Entity.model(name: node.identifier.text,
+                                       typeName: t,
+                                       acl: current?.acl,
+                                       overrideTypes: current?.overrides,
+                                       offset: node.offset,
+                                       length: node.length,
+                                       modelDescription: node.description,
+                                       processed: current?.isProcessed ?? false) {
+            current?.members.append(patmodel)
+        }
+        return .visitChildren
+    }
+    
+    func visit(_ node: InitializerDeclSyntax) -> SyntaxVisitorContinueKind {
+        current?.hasInit = true
+        if let initmodel = Entity.model(name: "init",
+                                        typeName: "",
+                                        acl: current?.acl,
+                                        throwsOrRethrows: node.throwsOrRethrowsKeyword?.text,
+                                        isStatic: false,
+                                        isInitializer: true,
+                                        offset: node.offset,
+                                        length: node.length,
+                                        modelDescription: node.description,
+                                        processed: current?.isProcessed ?? false) {
+            current?.members.append(initmodel)
+        }
+        return .visitChildren
+    }
+    
+    func visit(_ node: FunctionDeclSyntax) -> SyntaxVisitorContinueKind {
+        var isStatic = false
+        if let mds = node.modifiers {
```

Comment:
this `if` condition can be refactored into a separate function and used for this visit as well as for visiting `VarDeclSyntax`.

Reply:
ditto ^